src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import core.basesyntax.models.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
```

Comment:
<img width="890" alt="image" src="https://user-images.githubusercontent.com/96740815/192206462-86ffde08-1bc1-40c9-9914-a8b78bbffc62.png">

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/96740815/192206218-c0e709ba-9d12-4a08-986f-1e7eae63e40a.png">

Be attentive to common mistakes